db/migrate/20170410135556_add_csv_to_imports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCsvToImports < ActiveRecord::Migration[5.0]
+  def change
+    add_column :imports, :csv, :string
+    remove_column :imports, :uri
```

Comment:
Yeah, that API works out. If we were to start storing files on the server it would also look good: `File.open(import.file)`. Not that that's on the table!